docs/get-started.md

```diff
@@ -194,12 +194,16 @@ Then modify the webpack config to use it as a plugin:
 
 ```js
 // config/webpack/webpack.config.js
-const { webpackConfig, merge } = require("shakapacker");
-const ForkTSCheckerWebpackPlugin = require("fork-ts-checker-webpack-plugin");
+const { generateWebpackConfig, merge } = require('shakapacker')
+const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
 
-module.exports = merge(webpackConfig, {
-  plugins: [new ForkTSCheckerWebpackPlugin()],
-});
+const webpackConfig = generateWebpackConfig()
+
+module.exports = merge(
```

Comment:
not blocking, but fwiw this can be simplified further as `generateWebpackConfig` now takes a config to be merged in:

```
module.exports = generateWebpackConfig({
    plugins: [new ForkTsCheckerWebpackPlugin()]
});
```